screens/Messages/MessagesList.jsx

```diff
@@ -13,6 +13,7 @@ import styles from './style';
 
 const MessagesList = () => {
   const [value, setValue] = useState('');
+  const [sendMessages, setSendMessages] = useState(false);
```

Comment:
it's not a good practice to store different types in the same variable. Here you store a boolean in one case and an object in another case. Remind me to discuss this tomorrow if you don't see any good solution here

Reply:
Okay, will discuss tomorrow.